tasks/selinux_load_module.yml

```diff
@@ -10,14 +10,19 @@
       copy:
         dest: "{{ tempdir.path }}"
         src: "{{ item.path }}"
+        mode: preserve
 
     - name: Install module with priority
-      command: "semodule -n -s {{ store }} -X {{ priority }} -i {{ tempdir.path }}/{{ item.path | basename }}"
+      command: >-
+        semodule -n -s {{ store }} -X {{ priority }} -i
+          {{ tempdir.path }}/{{ item.path | basename }}
```

Comment:
I understand this is reported by the lint, but it seem to make the code worse for reading/understanding. Could you please explain why it's better to in this case?

Reply:
> I understand this is reported by the lint, but it seem to make the code worse for reading/understanding. Could you please explain why it's better to in this case?

The background for this effort is that Galaxy and Automation Hub are becoming quite strict with linting of code imported there in collections.  Rather than just suppressing everything, and therefore suppressing actual problems, we are trying to address all of the issues reported by the linters.

The linter likes yaml lines to be under a certain length.  But we do have some room for adjustment if excessive wrapping makes the code unreadable.  I prefer putting such ignores in-line rather than globally in a .yamllint.yml file.

@nhosoi please put a yamllint line-length ignore here instead of the wrapping.